docs/tutorials/ruby-driver-create-client.txt

```diff
@@ -144,7 +160,13 @@ URI Options Conversions
      - ``:max_idle_time => Float``
 
    * - maxStalenessSeconds=Integer
-     - ``:max_staleness => Float``
+     - ``{ :read => { :max_staleness => Integer }}``
+     
+       If the maxStalenessSeconds URI option value is -1, the option is ignored.
+       If the option value is an integer, the Ruby option is set to the
+       specified value (but note that the value should be either 0 or greater
+       than 90, while accepted by the driver, may cause errors during operations).
```

Comment:
I'm a little confused by this. Which values may cause errors during operations?

Reply:
Clarified the language here.